tcf_website/static/base/bugform.js

```diff
@@ -1,55 +1,49 @@
 import { validateForm } from "../common/form.js";
+import { postToDiscord, sendEmail } from "../common/send_feedback.js";
 
-function submitForm() {
+function submit(event) {
     var form = document.getElementById("bugform");
     var valid = validateForm(form);
     if (valid === true) {
-        postToDiscord();
-        // close form after submit
-        $("#bugModal").modal("toggle");
-        $("#confirmationModal").modal("toggle");
-    }
-}
+        var url = window.location.href;
+        var email = $("#emailField").val();
+        var description = $("#descriptionField").val();
+        var categories = "";
+        for (var i = 1; i <= 4; i++) {
+            var id = "#category" + i;
+            if ($(id).is(":checked")) {
+                categories += "[" + $(id).val() + "]";
+            }
+        }
 
-function resetForm() {
-    var form = document.getElementById("bugform");
-    form.classList.remove("was-validated");
-    var emailField = document.getElementById("emailField");
-    emailField.value = "";
-    var descriptionField = document.getElementById("descriptionField");
-    descriptionField.value = "";
+        var content = "Bug Found! \n**URL:** " + url +
+        "\n**Description:** \n" + description +
+        "\n**Categories:**" + categories +
+        "\n**Email:** " + email;
+        postToDiscord("bug", content);
 
-    for (var i = 1; i <= 4; i++) {
-        var id = "#category" + i;
-        $(id).prop("checked", false);
-    }
-}
+        content = "Bug Found! \n URL: " + url +
+        "\nDescription: \n" + description +
+        "\nCategories: " + categories +
+        "\nEmail: " + email;
+        sendEmail("Bug Report", content, "support@thecourseforum.com");
 
-function postToDiscord() {
-    var url = window.location.href;
-    var email = $("#emailField").val();
-    var description = $("#descriptionField").val();
-    var categories = "";
-    for (var i = 1; i <= 4; i++) {
-        var id = "#category" + i;
-        if ($(id).is(":checked")) {
-            categories += "[" + $(id).val() + "]";
-        }
+        content = "Thanks for reaching out! We received the following bug report from you:" +
+        "\n\nDescription: \n" + description +
+        "\nCategories:" + categories +
+        "\n\nWe apologize for any inconveniences that this may have caused." +
+        "Our team will be investigating the issue and will follow up with you shortly." +
+        "\n\nBest, \ntheCourseForum Team";
+        sendEmail("[theCourseForum] Thank you for your feedback!", content, email);
```

Comment:
Could you make these strings more readable?
https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript